### DISCUSSION TITLE
Globally show initials for users who don't have a custom gravatar

### DISCUSSION THREAD

**Discussion by mmandrus** [grafana/grafana#47359] [General]
Upon completion of [grafana-enterprise#544](https://github.com/grafana/grafana-enterprise/issues/544) ([corresponding PR](https://github.com/grafana/grafana/pull/47212) for those without GE access), @evictorero and I had a discussion about the value of default/fallback gravatars in general. Because the fallback images are random, they don't provide a good at-a-glance description of which user profile you are looking at. When gravatars are turned off entirely via config, every profile defaults to the same plain gray image, exacerbating the problem.

Our thought was that the mixing of custom gravatars + initials should be a global change, and that we should phase out the randomized fallback gravatars and blank profile images completely. If there is still a valid reason to keep fallback gravatars around, this could just apply to the situation where gravatars are disabled via config.

This could later be augmented by [grafana-enterprise#616](https://github.com/grafana/grafana-enterprise/issues/616), which would allow enterprise users to override initials with custom profile images without having to rely on third-party gravatars at all.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
